### PR TITLE
Added --prune-over-size-limit information to oadm prune images

### PR DESCRIPTION
**Pull request by soltysh**
This is part of this trello: https://trello.com/c/L9Coo2Zi.
The functionality already merged in origin: https://github.com/openshift/origin/pull/9481/

@miminar @adellape ptal

### CODE REVIEW

File: admin_guide/pruning_resources.adoc [hunk 1]
```diff
@@ -207,21 +212,46 @@ prune:
...
 
 * When an image is pruned, all references to the image are removed from all
 image streams that have a reference to the image in `*status.tags*`.
 
 * Image layers that are no longer referenced by any images are removed as well.
 
+[NOTE]
+====
+`--prune-over-size-limit` cannot be combined with `--keep-tag-revisions` nor
+`--keep-younger-than` flags. Doing so, will return an information that this
```
Comment (ahardin-rh) [inline admin_guide/pruning_resources.adoc]: drop the comma after so

File: admin_guide/pruning_resources.adoc [hunk 2]
```diff
@@ -148,8 +148,8 @@ $ oadm prune builds --orphans --keep-complete=5 --keep-failed=1 \
 
 == Pruning Images
 
-In order to prune images that are no longer required by the system due to age
-and status, administrators may run the following command:
+In order to prune images that are no longer required by the system due to age,
+status or exceed limits administrators may run the following command:
```
Comment (ahardin-rh) [inline admin_guide/pruning_resources.adoc]: comma after status
comma after limits

**Comment by ahardin-rh** [comment]
@soltysh just a few minor comments from me. Thanks!

File: admin_guide/pruning_resources.adoc [hunk 3]
```diff
@@ -207,21 +212,46 @@ prune:
...
+operation is not allowed.
+====
+
 To see what a pruning operation would delete:
 
 ----
 $ oadm prune images --keep-tag-revisions=3 --keep-younger-than=60m
 ----
 
+----
```
Comment (miminar) [inline admin_guide/pruning_resources.adoc]: Could you provide some description here as well? Or perhaps just merge it with the above command.

**Comment by soltysh** [comment]
Update ptal.

**Comment by ahardin-rh** [comment]
@soltysh Looks good! Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/pruning_resources.adoc #admin-guide-pruning-resources[Pruning Objects]
|Added --prune-over-size-limit information.
%

**Pull request merged**